web/themes/custom/votegov/src/sass/_variables.scss

```diff
@@ -6,9 +6,9 @@ $base-primary-hover: #2E598A; // blue-60
 $base-primary-active: $base-dark;
 
 // Accent Colors.
-$ac-warm: #CB5154; // red-cool-50
-$ac-warm-hover: #A13032;
-$ac-warm-active: #721113;
+$ac-warm: #992E30;
+$ac-warm-hover: #6D1516;
+$ac-warm-active: #6D1516;
```

Comment:
delete the `$ac-warm-active` color variable and replace any use of it in scss with `$ac-warm-hover`